internal/mount/flag.go

```diff
@@ -12,28 +12,41 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Helper functions for dealing with mount(8)-style flags.
 package mount
 
 import "strings"
 
-// Parse an option string in the format accepted by mount(8) and generated for
-// its external mount helpers.
+type ClientProtocol string
+
+const (
+	HTTP1 ClientProtocol = "http1"
```

Comment:
I know enums are not directly supported in golang. But can we still use the indirect enum type like this: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/fs/inode/core.go#L26.
So its more clear what values are expected.

Reply:
this is similar to the one that you pointed. The only difference is that it is a string enum. 
We can create an int enum and that would require another helper method similar to https://github.com/GoogleCloudPlatform/gcsfuse/blob/421afc5d40dda3976275669f33be2e50f0db550e/internal/fs/inode/core.go#L53 to convert the string value passed in flag into the respective int value of enum element. 
Please let me know if there's any concern around using string enum.